pipeline/src/lib.rs

```diff
@@ -79,25 +78,20 @@ pub fn serde_data_to_query_callback<T: FieldElement, S: serde::Serialize + Send
         }
     }
 }
+
 #[allow(clippy::print_stdout)]
 pub fn inputs_to_query_callback<T: FieldElement>(
-    inputs: HashMap<u32, Vec<T>>,
+    //inputs: HashMap<u32, Vec<T>>,
+    inputs: Vec<T>,
 ) -> impl QueryCallback<T> {
     move |query: &str| -> Result<Option<T>, String> {
         // TODO In the future, when match statements need to be exhaustive,
         // This function probably gets an Option as argument and it should
         // answer None by Ok(None).
 
         match &parse_query(query)?[..] {
-            ["\"input\"", index] => access_element("prover inputs", &inputs[&0], index),
-            ["\"data\"", index, what] => {
-                let what = what
-                    .parse::<u32>()
-                    .map_err(|e| format!("Error parsing what: {e})"))?;
-                assert_eq!(what, 0);
-
-                access_element("prover inputs", &inputs[&what], index)
-            }
+            //["\"input\"", index] => access_element("prover inputs", &inputs[&0], index),
```

Comment:
What about this one (and the parameter above)?

Reply:
gone